sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/ServerCallAsyncLiveTests.java

```diff
@@ -103,7 +103,7 @@ public void runAllClientFunctionsWithResponseAsync(HttpClient httpClient) {
             serverCallAsync = callingServerAsyncClient.initializeServerCall(groupId);
 
             Response<StartCallRecordingResult> startRecordingResponse =
-                serverCallAsync.startRecordingWithResponse(CALLBACK_URI).block();
+                serverCallAsync.startRecordingWithResponse(CALLBACK_URI, null, null, null, null).block();
```

Comment:
are you adding tests where the null values are actual values?

Reply:
yes, thats added :)